src/style.css

```diff
@@ -1,3 +1,93 @@
+:root {
+  --color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Using the universal selector `*` for resetting box-sizing can impact performance. Consider using more specific selectors to avoid potential issues.